### PR TITLE
[MRG] tidy up acqparserfif + example

### PR DESCRIPTION
**Pull request by jjnurminen**
Some minor tweaks:

-remove unnecessary one-line class method
-remove irrelevant comment
-simplify example (no need to instantiate the class explicitly)

### CODE REVIEW

File: examples/io/plot_elekta_epochs.py [hunk 1]
```diff
@@ -15,35 +15,34 @@
...
 ###############################################################################
-# Read raw file and create parser instance
+# Read raw file
 raw = mne.io.read_raw_fif(fname_raw)
-ap = AcqParserFIF(raw.info)
 
 ###############################################################################
 # Check DACQ defined averaging categories and other info
-print(ap)
+print(raw.acqparser)
```
Comment (agramfort) [inline examples/io/plot_elekta_epochs.py]: I don't like this as everytime you do: raw.acqparser
you call a property and redo the same computation (create instance, parse info etc.)

Comment (Eric89GXL) [inline examples/io/plot_elekta_epochs.py]: that isn't how it works -- it's created once when accessed the first time

https://github.com/mne-tools/mne-python/blob/master/mne/io/fiff/raw.py#L424

**Comment by jjnurminen** [comment]
Hooray for property descriptor magic ;)

**Comment by agramfort** [comment]
arfff ok ...

**Comment by jjnurminen** [comment]
@agramfort I know you don't like magic, but this is very beneficial white magic, no harmful side effects.

**Comment by Eric89GXL** [comment]
Thanks @jjnurminen

**Pull request merged**